test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -784,7 +784,8 @@ class test_uct_cm_sockaddr : public uct_test {
             std::string err_str = format_message(message, ap);
             if ((strstr(err_str.c_str(), "client: got error event RDMA_CM_EVENT_ADDR_ERROR"))  ||
                 (strstr(err_str.c_str(), "client: got error event RDMA_CM_EVENT_ROUTE_ERROR")) ||
-                (strstr(err_str.c_str(), "rdma_resolve_route(to addr=240.0.0.0"))) {
+                (strstr(err_str.c_str(), "rdma_resolve_route(to addr=240.0.0.0")) ||
+                (strstr(err_str.c_str(), "error event on client ep"))) {
```

Comment:
maybe rework this way:
https://github.com/openucx/ucx/blob/7b231a30721102040bb66a56e83bf6d1c8870fa5/test/gtest/ucp/test_ucp_sockaddr.cc#L117
or even add a helper class for this?

Reply:
you mean only this function or unite all detect_loggers in this file?

Reply:
first - for this function and then take a look if helper class does makes sense to reuse it everywhere for logger wrappers

Reply:
the reason for splitting the detect_error functions is so that we can use each specific one only for the expected failure in a specific test. that's why i don't think we should unite all tests' failures to one function.
having a stop_list or keeping it the way it is now, sounds about the same. both seem clear. why is stop_list better?
anyway, if we change only this function to stop_list, i think would need to change others for consistency, and would be better to do in a separate pr.

Reply:
we should not unite lists, only logic. stop list would be a parameter
+1 for separate PR with consistency refactoring